study-builder/studies/circadia/sendgrid-emails.conf

```diff
@@ -0,0 +1,12 @@
+{
+  "sendgridEmails": [
+    {
+      "key": "welcome",
+      "name": "CGC_WELCOME",
```

Comment:
Is it `CGC` or `CRC`?

Reply:
@yufengwng `CRC` is  just shortened version of `CiRCadia`

Reply:
This email was replaced by one that is sent from Auth0 and it was removed [here](https://github.com/broadinstitute/ddp-study-server/pull/856).